js-exercises/args-string/argsString.js

```diff
@@ -0,0 +1,8 @@
+const argsString = (message, args) => {
```

Comment:
Can probably change the `args` variable name to something more descriptive.

Reply:
here the arguments are pretty generic that's why took the call to keep them as args in a generic way. Any better name suggestion ?